stripe/src/main/java/com/stripe/android/model/ShippingInformation.java

```diff
@@ -0,0 +1,78 @@
+package com.stripe.android.model;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.stripe.android.util.StripeNetworkUtils;
+
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.stripe.android.util.StripeJsonUtils.optString;
+import static com.stripe.android.util.StripeJsonUtils.putStringIfNotNull;
+
+/**
+ * Model representing a shipping address object
+ */
+public class ShippingInformation extends StripeJsonModel {
+
+    private static final String FIELD_ADDRESS = "address";
+    private static final String FIELD_NAME = "name";
+    private static final String FIELD_PHONE = "phone";
+
+    private @Nullable SourceAddress mAddress;
```

Comment:
I think we should unify on a single `Address` type, and use it everywhere. For the Card object, it might be worth adding an `Address` getter/setter too.